utils/tez.ts

```diff
@@ -0,0 +1,9 @@
+import BN from "bignumber.js";
+
+const MUTEZ_TEZ_RATIO = Math.pow(10, 6);
+
+export const mutezToTez = (mutez: number): number =>
```

Comment:
Can you add some unit test for these two functions?
and make CI run test when build?